### ISSUE TITLE
Fonts not working in `write_png` and `write_pdf`

### ISSUE THREAD

**Issue by a-maldet** [R-package/styledTables#12]
For the `write_pdf` and `write_png` it should be possible to add custom fonts. Therefore it should be possible to inject custom LaTeX-code in the preamble. 

Maybe `write_pdf <- function(x, resize_pdf, inject_latex_preamble = function() {....})`

**Comment by a-maldet** [comment]
@GregorDeCillia could you take over this issue?

**Comment by GregorDeCillia** [comment]
Yes, I could do that. How should a client use `ìnject_latex_preamble`? Is it just about passing latex packages and package options or should literal latex strings be supported as well?

One thing that could be tricky with literal strings are duplicates: Should they be recognized and removed automatically?

**Comment by GregorDeCillia** [comment]
@a-maldet Also, please provide a minimal example where the latex rendering fails so I can test everything properly.

**Comment by a-maldet** [comment]
Sorry, closed the wrong issue ... This issue is still open

**Comment by a-maldet** [comment]
Sorry, closed the wrong issue ... This issue is still open

**Comment by GregorDeCillia** [comment]
There is now a hidden feature called `inject_preamble()` in the package that allows to attach packages to styledtables objects. These packages are then used in `knit_print_rnw()` and `knit_print_rmd_latex()`. 

Usage:

```r
library(styledTables)
mtcars %>%
  styled_table() %>%
  inject_preamble(package = "multirow", options = NULL, extra_lines = "% some tex code")
```

Todos:

- [ ] use these extra packages also in `write_pdf`. This is also for `knit_print_rmd_html()`
- [ ] export and document this feature
- [ ] avoid duplication of preambles in case of `rnw` documents.
- [ ] add additional support for certain font types via a wrapper around `set_latex_font_name` that also loads all necessary packages automatically.
